core/protocols/aws-json-protocol/src/main/java/software/amazon/awssdk/protocols/json/BaseAwsJsonProtocolFactory.java

```diff
@@ -242,6 +243,18 @@ public final SubclassT protocolVersion(String protocolVersion) {
             return getSubclass();
         }
 
+        /**
+         * ContentType  of the client (By default it is used from {@link #AWS_JSON} ).
+         * Used to determine content type.
+         *
+         * @param contentTypeOverride JSON protocol contentTypeOverride.
+         * @return This builder for method chaining.
+         */
+        public final SubclassT contentTypeOverride(String contentTypeOverride) {
+            protocolMetadata.contentTypeOverride(contentTypeOverride);
+            return getSubclass();
+        }
```

Comment:
I think in the future we will move contentType to be always controlled from the codegen, because we'll need to treat each protocol differently. Should we just name this "contentType" since in the future it won't really be an "override"?

Reply:
Changed all occurrences of contentTypeOverride to contentType.